server/routes/objects.js

```diff
@@ -70,42 +75,35 @@ router.get('/:url', async (req, res) => {
         await Promise.all(
           objects.map(async name => {
             let tname = name.split('/')[3];
-            const uniqueId = name.split('png').pop();
-            await file_stats(user, bucketName, tname.split('.')[0]).then(
-              response => {
-                if (response.length > 0) {
-                  temp.push({
-                    name: tname.split('.')[0],
-                    format: response[0]?.file_type,
-                    date: response[0]?.upload_date,
-                    fileId: uniqueId,
-                    isUploaded: response[0]?.is_uploaded,
-                  });
-                }
-              },
-            );
+            await file_stats(bucketName, tname.split('.')[0]).then(response => {
+              if (response[0]?.isUploaded)
+                temp.push({
+                  name: tname.split('.')[0],
+                  format: response[0]?.file_type,
+                  date: response[0]?.upload_date,
+                });
+            });
```

Comment:
The error handling for the file upload should log the error using the logger instead of console.error. This ensures consistent logging throughout the application.
```diff
-        console.error(err);
+        log.error(err);
```
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*